src/components/AppHeader/AppHeader.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
+
+import { DispatchContex, StateContex } from '../../Store';
+import { Todo } from '../../types/Todo';
+
+export const AppHeader: React.FC = () => {
+  const [newTitle, setNewTitle] = useState('');
+  const dispatch = useContext(DispatchContex);
+  const { todos } = useContext(StateContex);
+
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (inputRef.current !== null) {
+      inputRef.current.focus();
+    }
+  }, [todos]);
+
+  const isAllCompleted =
+    todos.filter(todo => todo.completed).length === todos.length;
+
+  const handlerFormSubmit = (evt: React.FormEvent<HTMLFormElement>) => {
```

Comment:
Consider rename evt to event

Reply:
I like to name the evt. There were no problems before that